### ISSUE TITLE
No menus, help or shortcuts

### ISSUE THREAD

**Issue by drozzy** [CodeBrew-io/Issues#19]
Using this on Sep 9, 2014 the whole editor shows up as a blank page (only with the cog and a help icon) --- with no toolbars, no shortcuts help, nothing.

P.S.: I'm using Chrome, on Windows 8.1, 64bit.

Is there something missing?

**Comment by MasseGuillaume** [comment]
Yep it's like that since the last update. What feature do you miss ?

**Comment by drozzy** [comment]
Um.. how do I get it do anything? Is there help for shortcuts?
